app/main.ipynb

```diff
@@ -25,18 +25,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "import sys\n",
+    "import time\n",
```

Comment:
The 'time' library is imported but not used in the code. Consider removing it if it's not needed to keep the code clean.